index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./style.css" />
+    <title>PortfolioPage</title>
+    <link href='https://fonts.googleapis.com/css?family=Poppins' rel='stylesheet'>
+  </head>
+  <body>
+    
+    <main>
+      <header>
+        <div class="logo-placeholder">
+          <div class="logo"><a href="#">My Logo</a></div>
+          </div>
+        <nav class="menu"><img src="./logos/Union.png" alt="menu" /></nav>
+      </header>
+      <section class="container">
+        <div class="primary-text">I’m Richmond Glad to see you!</div>
+        <div class="content">
```

Comment:
- [ ] Please use `hx` tag here instead of `div` element `I'm Richmond Glad to see you` this is an introductory content

Reply:
@MeqdamAlqudah it report it cant be a child element